app/models/insured/factories/self_service_factory.rb

```diff
@@ -201,6 +203,14 @@ def calculate_max_applicable_aptc(enrollment, new_effective_on)
         end
       end
 
+      def calculate_max_tax_credit(enrollment, new_effective_on)
+        if EnrollRegistry.feature_enabled?(:temporary_configuration_enable_multi_tax_household_feature)
+          max_aptc = ::Operations::PremiumCredits::FindAptc.new.call({hbx_enrollment: enrollment, effective_on: new_effective_on}).value!
```

Comment:
we're calling this operation twice in the same method. move this operation call into it's own method and use memoization to return value second time.